docs/source/ide_basics.rst

```diff
@@ -23,7 +23,10 @@ You can open an IDE file like so::
     with idelib.importFile(filename) as ds:
         print(type(ds))
 
-Note: a :class:`Dataset` object perfoms *lazy-loading*, meaning that it only loads information as is needed. As a result, it internally retains a handle to the source file which after use needs to be closed. This can be accomplished by either using :class:`Dataset` as a *context manager* (as seen above; this is the recommended method), or by using :class:`Dataset` as a normal value and calling the ``close()`` method manually::
+Note: a :class:`Dataset` object perfoms *lazy-loading*, meaning that it only loads information as is needed. As a result, it internally retains a handle to the source file which **needs to be closed after use**. This can be accomplished by either:
```

Comment:
I'm not sure "needs to be closed" needs to be stressed. There are lots of things that have open files/streams; the stress makes it sound like some kind of dangerous variation. The file pointer will close when the object is deleted (like leaving scope). This is only problematic when writing, and only in some OS.

Reply:
sure, that's fair. addressed in 28a0a0a48626667fc3ee58b36291cfd0392a4461